pkg/machine/apple/apple.go

```diff
@@ -275,6 +276,42 @@ func StartGenericAppleVM(mc *vmconfigs.MachineConfig, cmdBinary string, bootload
 
 	logrus.Debugf("helper command-line: %v", cmd.Args)
 
+	if mc.LibKrunHypervisor != nil && logrus.IsLevelEnabled(logrus.DebugLevel) {
+		rtDir, err := mc.RuntimeDir()
+		if err != nil {
+			return nil, nil, err
+		}
+		kdFile, err := rtDir.AppendToNewVMFile("krunkit-debug.sh", nil)
+		if err != nil {
+			return nil, nil, err
+		}
+		f, err := os.Create(kdFile.Path)
+		if err != nil {
+			return nil, nil, err
+		}
+		err = os.Chmod(kdFile.Path, 0744)
+		if err != nil {
+			return nil, nil, err
+		}
+
+		_, err = f.WriteString("#!/bin/sh\nexec ")
+		if err != nil {
+			return nil, nil, err
+		}
+		for _, arg := range cmd.Args {
+			_, err = f.WriteString(fmt.Sprintf("%q ", arg))
+			if err != nil {
+				return nil, nil, err
+			}
+		}
+		err = f.Close()
+		if err != nil {
+			return nil, nil, err
+		}
+
+		cmd = exec.Command("/usr/bin/open", "-Wa", "Terminal", kdFile.Path)
```

Comment:
i assume this will exist on all Mac installations?

Reply:
Yes, it's part of the base OS: https://www.unix.com/man-page/osx/1/open/